lib/generators/solid_cache/install/install_generator.rb

```diff
@@ -3,22 +3,70 @@
 class SolidCache::InstallGenerator < Rails::Generators::Base
   source_root File.expand_path("templates", __dir__)
 
-  class_option :skip_migrations,    type: :boolean, default: nil,
-                                    desc: "Skip migrations"
-
   def add_rails_cache
-    if (env_config = Pathname(destination_root).join("config/environments/production.rb")).exist?
-      gsub_file env_config, /(# )?config\.cache_store = (:.*)/, "config.cache_store = :solid_cache_store"
-    end
+    gsub_file app_root.join("config/environments/production.rb"),
+      /(# )?config\.cache_store = (:.*)/, "config.cache_store = :solid_cache_store"
   end
 
   def create_config_solid_cache_yml
     template "config/solid_cache.yml"
   end
 
-  def create_migrations
-    unless options[:skip_migrations]
-      rails_command "railties:install:migrations FROM=solid_cache", inline: true
+  def add_cache_db_to_database_yml
+    if app_is_using_sqlite?
+      gsub_file database_yml, /production:\s*<<: \*default.*/m, sqlite_database_config_with_cache
+    else
+      gsub_file database_yml, /production:\s*<<: \*default.*/m, generic_database_config_with_cache
```

Comment:
Not sure how you want to handle this situation, but if someone is adding Solid Cache to an application where they have already added, for example, Solid Queue—and thus their `database.yml` is already 3-tier, this won't add the cache database to their config.

Reply:
Yes, this needs to be more resilient. If you're interested, please do take a look to make it so! We want Solid Cache and Solid Queue configured out of the box and with sqlite by default, so this does need to be resolved.

Reply:
Going to release with this for now to move the ball forward on integration. But we will have to revisit this again when we have a fix so we can get solid queue in on the same deal.